src/components/transcript/transcript.scss

```diff
@@ -160,4 +167,9 @@
 
 .autoscroll-button-visible {
   opacity: 1;
-}
\ No newline at end of file
+}
+
+:global(.playkit-player .playkit-top-bar .playkit-right-controls) {
```

Comment:
you have duplication of the styles here:
https://github.com/kaltura/playkit-js-transcript/pull/98/files#diff-b91f3ed9694478c752ed5d1e141b1bb7346db9246edca21bb731562928a5aef6R19-R21